### PR TITLE
Add golang and .Net to the ootb language subcategories

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://github.com/openshift/origin-web-catalog/issues/259

### CODE REVIEW

File: dist/origin-web-catalogs.js [hunk 1]
```diff
@@ -213,6 +218,11 @@ webpackJsonp([ 0, 1 ], [ function(e, t) {
                 label: "Python",
                 tags: [ "python" ],
                 icon: "font-icon icon-python"
+            }, {
+                id: "golang",
+                label: "Go",
+                tags: [ "Golang", "golang", "go" ],
```
Comment (jwforres) [inline dist/origin-web-catalogs.js]: included Golang with capital G for backwards compatibility with the the old categories

Comment (spadgett) [inline dist/origin-web-catalogs.js]: I think the old catalog tag checking was case insensitive (but I could be wrong).

Comment (spadgett) [inline dist/origin-web-catalogs.js]: https://github.com/openshift/origin-web-console/blob/master/app/scripts/services/catalog.js#L28

Comment (jwforres) [inline dist/origin-web-catalogs.js]: ok i'll double check what it was doing

**Comment by jwforres** [comment]
@spadgett updated to make tag matching case insensitive

**Pull request merged**